src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 5;
+    private static final int PARAM_BOUND = 25;
```

Comment:
max length?

Reply:
I will change the names of these two variables to be more readable if this is what you mean.